app/controllers/pieces_controller.rb

```diff
@@ -7,6 +7,7 @@ def update
     else
       render text: 'invalid move'
     end
+    @piece.game.whose_turn(@piece.user) # switch players after turn
```

Comment:
I don't think this will actually work, since `whose_turn` is expecting a color name rather than a user object.  But that should be pretty easy, since you can do `@piece.color` instead!

Reply:
ugh of course! m-)

Reply:
Ugh of course! Thank you!